<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RUNTEST</class>
 <widget class="QWidget" name="RUNTEST">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1101</width>
     <height>621</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0" rowminimumheight="0,50,0,0,0,0,0,0,0,0" columnminimumwidth="0,0,350,350">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>7</number>
    </property>
    <item row="4" column="0">
     <widget class="QListView" name="xlistView"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="Plotting_Label">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X Axis</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QListView" name="ylistView"/>
    </item>
    <item row="8" column="0" rowspan="2" colspan="2">
     <widget class="QLineEdit" name="sampleRateEdit"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sampling Rate (Points/sec)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y Axis</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="EndDCButton">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>End Data Collection</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLCDNumber" name="NumDPlcd">
      <property name="digitCount">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="OpenAFRTableButton">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Open AFR TABLE</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="NumDPLabel">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Number of Data Points Collected:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="PlotDataButton">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Plot Data</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QSlider" name="sampleRateSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="serialPortNameLabel">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NONE</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="8" colspan="2">
     <widget class="QTextBrowser" name="DataBrowser">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QPushButton" name="StartDCButton">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start Data Collection</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="setSerialPortButton">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Set Serial Port:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
